pkg/kots/lint.go

```diff
@@ -140,6 +147,15 @@ func LintSpecFiles(specFiles SpecFiles) ([]LintExpression, bool, error) {
 		return renderContentLintExpressions, false, nil
 	}
 
+	targetMinLintExpressions, err := lintTargetMinVersions(filteredFiles)
+	if err != nil {
+		return nil, false, errors.Wrap(err, "failed to lint render content")
```

Comment:
we shouldn't error out here since this function depends on an external api and we don't want the linting service to go down if that api is down or unstable. let's just log the error.